pantalaimon/daemon.py

```diff
@@ -455,6 +455,10 @@ def sanitize_filter(self, sync_filter):
         path = request.path
         method = request.method
 
+        # This is very much a dirty hack, but it is fine for now
+        if "#" in path:
+            path = path.replace("#", "%23")
```

Comment:
Can't we use the  [quote](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote) method from urllib here?

Reply:
oooo, yes - we should do that. I was worried that it would convert `/` but it says it doesn't. Will try it.